Android/Java/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.java

```diff
@@ -171,15 +214,14 @@ public void destroyBannerButtonTapped(View view) {
     }
 
     private void destroyBanner() {
-        if (bannerParentLayout != null) {
+        if (bannerParentLayout != null && mBannerAd != null) {
```

Comment:
I wrote this here because I can’t write a comment close to the function:
As I see it, the 'destroyBannerButtonTapped' function still exists, and I think we can remove it. If we do that, this function won't be in use and can be removed as well.